src/core/DOMEval.js

```diff
@@ -3,12 +3,19 @@ define( [
 ], function( document ) {
 	"use strict";
 
-	function DOMEval( code, doc ) {
+	function DOMEval( code, doc, type, src ) {
 		doc = doc || document;
 
 		var script = doc.createElement( "script" );
-
-		script.text = code;
+		if ( code ) {
+			script.text = code;
+		}
+		if ( type === "module" ) {
```

Comment:
The HTML Standard [specifies](https://html.spec.whatwg.org/#attr-script-type) a case-insensitive match for "module", which we need to respect.